cmd/lifecycle.go

```diff
@@ -157,12 +145,7 @@ func (sys *LifecycleSys) Init(objAPI ObjectLayer) error {
 }
 
 // Refresh LifecycleSys.
-func (sys *LifecycleSys) refresh(objAPI ObjectLayer) error {
-	buckets, err := objAPI.ListBuckets(context.Background())
```

Comment:
I think we can do this way:
```go
if buckets == nil {
        buckets, err = objAPI.ListBuckets(context.Background())
	if err != nil {	
		logger.LogIf(context.Background(), err)	
		return err	
  	}
        sys.removeDeletedBuckets(buckets)
}
```

And set buckets to nil in the toplevel after consuming buckets the first time

Reply:
Oh, you are confusing this with something else, this refresh is not used in some go-routine anymore. Because any updates to lifecycle, policy or notification are communicated over peers and they update their relevant configs as needed.

It is not needed to add a long-running go-routine which continuously listBuckets()

NOTE: sweepRound and lifecycleRound are not touched